### ISSUE TITLE
Not launching in Korean environment with default `language: 'auto'` config

### ISSUE THREAD

**Issue by floydkim** [jesseduffield/lazygit#3743]
**Describe the bug**

After upgrading to version 0.43.0, lazygit does not launch with the default language setting ‘auto’.

I have resolved the issue temporarily by setting the language to 'ko' or 'en' in the config file as follows:

```yaml
gui:
    language: ko
```
(config file : `~/Library/Application\ Support/lazygit/config.yml`)

**To Reproduce**

1. Install lazygit `0.43.0` (using homebrew)
2. Open a terminal and run lazygit.

error
```bash
2024/07/13 16:35:26 open translations/ko-KR.json: file does not exist
```

**Expected behavior**

lazygit should launch.

**Screenshots**

<img width="1097" alt="image" src="https://github.com/user-attachments/assets/a3a2dd83-30b3-48f0-a6a4-6ea0826fefa1">

**Version info:**
_Run `lazygit --version` and paste the result here_
```
commit=, build date=, build source=homebrew, version=0.43.0, os=darwin, arch=arm64, git version=2.39.3 (Apple Git-146)
```
_Run `git --version` and paste the result here_
```
git version 2.39.3 (Apple Git-146)
```

**Additional context**

**Note:** please try updating to the latest version or [manually building](https://github.com/jesseduffield/lazygit/#manual) the latest `master` to see if the issue still occurs.

**Comment by stefanhaller** [comment]
Thanks for the report, what a stupid bug. Here's a fix: #3744

**Comment by floydkim** [comment]
it's fixed and working well. thank you so much 😄 @stefanhaller
